Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,94 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+from mentii import user_ctrl
+
+class MentiiAuthentication:
```

Comment:
It's functionally equivalent but static methods give us a little extra. Because this is created as a class we could create a sub class that would change the way those methods work. For this example it probably doesn't make sense to do that though because subclassing MentiiAuthentication just doesn't sound like something we'll want to do. Top level functions (just a module) can't be overwritten like that. 

For our purposes it's mostly a style thing. If each of our util files are python modules rather than classes we'll be importing them and accessing them all the same way.